services/datamanager/builtin/sync/file_deletion_windows.go

```diff
@@ -0,0 +1,47 @@
+package sync
+
+import (
+	"context"
+
+	"github.com/benbjohnson/clock"
+	"go.viam.com/rdk/logging"
+	"go.viam.com/rdk/utils/diskusage"
+)
+
+func maybeDeleteExcessFiles(
```

Comment:
This is never going to do any work right?

I'd recommend having this just do nothing and upstream here:
https://github.com/viamrobotics/rdk/pull/4603/files#diff-8c92f604aee5dda6406d17054cce5fb6a1fa01d3546f58cd598d241fbd3de978R36

early return if the feature is running on android or windows.

Reply:
> never going to do any work right

yup, it's a no-op for now

it needs its own file because the `diskusage.Statfs` call doesn't exist on windows -- we can't just guard it with an if statement

could implement Statfs for windows as fast-follow